ulid.go

```diff
@@ -340,3 +341,24 @@ func (id *ULID) SetEntropy(e []byte) error {
 func (id ULID) Compare(other ULID) int {
 	return bytes.Compare(id[:], other[:])
 }
+
+// Scan implements the sql.Scanner interface.
+func (id *ULID) Scan(src interface{}) error {
+	switch x := src.(type) {
+	case nil:
+		return nil
+
+	case string:
+		return id.UnmarshalText([]byte(x))
+
+	case []byte:
+		return id.UnmarshalBinary(x)
+	}
+
+	return errors.New("ulid: source value must be bytes")
+}
+
+// Value implements the sql/driver.Valuer interface.
```

Comment:
Let's add clear documentation in this comment that the generated value is of the []byte type and how to do it with strings via the wrapper type approach.